Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -23,47 +23,85 @@ don't interact with those.
 In order to be able to draw an image to a screen, we first have to provide some information to
 Splashkit. We have to have Bitmap images to pass into our
 [Create Sprite](https://splashkit.io/api/sprites/#create-sprite) function so that it can create a
-workable _Sprite_ file type for us. This looks something like this inside our Main() method:
-  
-```bash
-Private Bitmap happySun;
-Private Sprite happySunSprite;
-```
+workable _Sprite_ file type for us.
 
 Line by line, this can be broken down as:
 
 1. Instantiate a new Object happySun, of Type [Bitmap](https://splashkit.io/api/types/#bitmap)
 
-```csharp
+<Tabs>
+  <TabItem label="C#">
+  ```csharp
 Bitmap happySun;
+Sprite happySunSprite;
 ```
+  </TabItem>
+
+  <TabItem label="C++">
+  ```c++
+    bitmap HappySun = load_bitmap("Happy Sun", "sprite01.png");
+   ```
+  </TabItem>
+
+</Tabs> 
 
 Relevant overloads are shown below, we pass in a string name, and we can either give a filename to
 assign from a file or URL we have access to, or denote an int for width and height to have a
 workable 'empty' sprite.
 
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 Bitmap(string name, string file);
 ```
-
 ```csharp
 Bitmap(string name, int width int height);
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+public static Bitmap SplashKit.CreateBitmap(string name, int width, int height);
+public Bitmap(string name, int width, int height);
```

Comment:
This is C# code. I think you want: `bitmap load_bitmap(string name, string filename)` (https://splashkit.io/api/graphics/#load-bitmap) and `bitmap create_bitmap(string name, int width, int height)` (https://splashkit.io/api/graphics/#create-bitmap)